netsage_cms/src/taccsite_cms/settings_custom.py

```diff
@@ -1,37 +1,14 @@
 # CUSTOM SETTINGS VALUES.
-# TACC WMA CMS SITE:
+# TACC WMA (SAD) CMS SITE:
 # *.NETSAGE.TACC.UTEXAS.EDU
 
-########################
-# DJANGO (EMAIL)
-########################
-
-# Set on server, NOT here
-# https://tacc-main.atlassian.net/wiki/x/ZhVv
-# EMAIL_BACKEND = "..."
-# EMAIL_HOST = "..."
-# DEFAULT_FROM_EMAIL = "..."
-
-########################
-# DJANGO CMS SETTINGS
-########################
-
-CMS_TEMPLATES = (
-    ('standard.html', 'Standard'),
-    ('fullwidth.html', 'Full Width'),
-
-    ('guide.html', 'Guide'),
-    ('guides/getting_started.tam.html', 'Guide: Getting Started'),
-    ('guides/data_transfer.html', 'Guide: Data Transfer'),
-    ('guides/data_transfer.globus.html', 'Guide: Globus Data Transfer'),
-    ('guides/portal_technology.html', 'Guide: Portal Technology Stack'),
-)
```

Comment:
**Why?** Netsage does not use any templates beyond the defaults (it even uses less).